Composer/packages/client/src/components/AddRemoteSkillModal/SkillDetail.tsx

```diff
@@ -0,0 +1,71 @@
+/* eslint-disable format-message/literal-pattern */
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+/** @jsx jsx */
+import { jsx, css } from '@emotion/core';
+import React from 'react';
+import formatMessage from 'format-message';
+interface SkillDetailProps {
+  manifest: {
+    dispatchModels: {
+      intents: Array<string> | object;

```

Comment:
string[]

Reply:
your type does not indicate that intents or languages under dispatchModels can be optional (have undefined values) but everywhere you try to access them you are doing ```?.```, if that's the case, make them optional in your type.